src/renderer/src/views/Migration/index.tsx

```diff
@@ -138,15 +147,41 @@ const messages = defineMessages({
   migratePreviousInstruction: {
     id: 'views.migration.migratePreviousInstruction',
     defaultMessage:
-      'We have found a previous Mapeo project. Follow the steps below to begin migrating to the new Mapeo:',
+      'We have found a previous Mapeo project. To begin using New Mapeo with the data you have on old Mapeo, data migration is required. Once you are ready to migrate your data to new Mapeo, follow the steps below.',
```

Comment:
not grammatical so feel free to ignore:

```suggestion
      'We have found a previous Mapeo project. To begin using MapeoSuperS with the data you have on Mapeo 5, data migration is required. Once you are ready to migrate your data to MapeoSuperS, follow the steps below.',
```